client/src/components/molecules/MyDailyTimeRecordTable/DesktopTable.tsx

```diff
@@ -47,40 +48,43 @@ const DesktopTable: FC<Props> = ({ table, isLoading }): JSX.Element => {
         )}
       >
         <>
-          {isLoading ? (
-            <TableSkeleton rows={11} column={10} />
+          {error == null ? (
+            isLoading ? (
```

Comment:
```suggestion
          {error === null ? (
            isLoading ? (
```

Check if this code is working, if not refer to the first option
```suggestion
          {error ? (
            isLoading ? (
```